lib/enrichment/src/find_enrichment_table_records.rs

```diff
@@ -93,13 +114,18 @@ impl Expression for FindEnrichmentTableRecordsFn {
         &mut self,
         state: &mut state::Compiler,
     ) -> std::result::Result<(), ExpressionError> {
-        self.index = Some(add_index(state, &self.table, &self.condition)?);
+        self.index = Some(add_index(
+            state,
+            &self.table,
+            self.case_sensitive,
+            &self.condition,
+        )?);
         Ok(())
     }
 
     fn type_def(&self, _: &state::Compiler) -> TypeDef {
         TypeDef::new()
-            .fallible()
+            .infallible()
```

Comment:
Is this change related? Or just something you noticed?

Reply:
Strange. That change should not be there.

I think in theory we should be able to make this function infallible, but it's not quite there yet.